shared/metadata_test.go

```diff
@@ -321,3 +330,48 @@ func TestGetMetadataFilePath(t *testing.T) {
 	actual = GetMetadataFilePath("foo")
 	assert.Equal(t, "foo/META.yml", actual)
 }
+
+func TestPrepareLinkFilter(t *testing.T) {
+	label := "labelA"
+	subtestName := "Something should happen"
+	fail := TestStatusFail
+	metadataResults := map[string]MetadataLinks{
+		"/foo/bar/a.html": []MetadataLink{
+			{
+				URL: "https://bug.com/item",
+				Results: []MetadataTestResult{{
+					SubtestName: &subtestName,
+					Status:      &fail,
+					Label:       &label,
```

Comment:
I'm not too sure what this test is about, but can subtests really have an associated label? I guess it comes for free by copying the link metadata, but wouldn't it also mean that all subtests will be filtered out when searching for a label, because none on the subtests have that label?

Reply:
That is correct. Technically this format won't exist because label is at the test level. Let me change that

Reply:
Subtests cannot be associated with a test-level test but this test is for `TestPrepareLinkFilter`.  This is a drive-by change. See `TestPrepareTestLabelFilter` below

Reply:
Actually you are right. Fixed it. Labels should be at the test-level